distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Dialect.enso

```diff
@@ -567,6 +568,10 @@ ends_with = Base_Generator.lift_binary_op "ENDS_WITH" str-> sub->
     res = str ++ " LIKE CONCAT('%', " ++ sub ++ ")"
     res.paren
 
+## PRIVATE
+regex_match = Base_Generator.lift_binary_op "REGEX_MATCH" str-> pattern->
```

Comment:
any reason why not:
```suggestion
private _regex_match = Base_Generator.lift_binary_op "REGEX_MATCH" str-> pattern->
```

Don't think we share outside the file (and shouldn't if we do).

Reply:
I'm going to fix this in a follow up MR. I want to change it, but I also want consistency so I'll do a refactor MR and get them all.